### PR TITLE
Rename "openshift-ansible-aws-rhel" to "aws-centos" and add release-3.11 job

### PR DESCRIPTION
**Pull request by vrutkovs**
A followup for https://github.com/openshift/release/pull/1524 - the profile is 
renamed and a new job added to run this on release-3.11.

Added OWNERS file to make sure openshift-ansible can update this profile

This also includes a fix for correct AWS key name

Current status:
* [x] Tests start, but too many fail. Probably VM is too slow or misconfigured

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Run this:

```
#!/usr/bin/env python

import sys
import yaml

print sys.argv[1]


data = {}
edited = False
with open(sys.argv[1]) as f:
	data = yaml.load(f)
	for job_type in data:
		if job_type == "periodics":
			continue

		for repo in data[job_type]:
			for job in data[job_type][repo]:
				if job["agent"] != "kubernetes":
					continue

				if job["spec"]["containers"][0]["command"][0] != "ci-operator":
					continue

				target = "all"
				for arg in job["spec"]["containers"][0].get("args", []) + job["spec"]["containers"][0]["command"]:
					if arg.startswith("--target="):
						target = arg[len("--target="):].strip("[]")
						break

				branch = "master"
				if "branches" in job:
					branch = job["branches"][0]

				prefix = "pull"
				if job_type == "postsubmits":
					prefix = "branch"

				name = "{}-ci-{}-{}-{}".format(prefix, repo.replace("/", "-"), branch, target)
				job["name"] = name
				edited = True

if edited:
	with open(sys.argv[1], 'w') as f:
		f.write(yaml.dump(data))
```

**Comment by vrutkovs** [comment]
Done, PTAL

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml [hunk 1]
```diff
@@ -72,6 +72,61 @@ presubmits:
           - configMap:
               name: cluster-profile-gcp
     trigger: ((?m)^/test( all| gcp-major-upgrade),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - release-3.11
+    - dockerless-build-temporary
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml]: Don't cargo-cult this, please -- I think it's out of date anyway

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml [hunk 2]
```diff
@@ -72,6 +72,61 @@ presubmits:
...
+    context: ci/prow/aws
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml]: This should be `e2e-aws`

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml [hunk 3]
```diff
@@ -72,6 +72,61 @@ presubmits:
...
+    decorate: true
+    name: pull-ci-openshift-openshift-ansible-release-3.11-e2e-aws
+    rerun_command: /test aws
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml]: This should be `e2e-aws`

Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml]: Fixed

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml (hunk 2 above)
Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml]: Fixed

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml [hunk 4]
```diff
@@ -72,6 +72,61 @@ presubmits:
...
+          value: e2e-aws
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: master.yaml
+              name: ci-operator-openshift-openshift-ansible
+        - name: RPM_REPO
+          value: https://rpms.svc.ci.openshift.org/openshift-origin-v3.11/
```
[the remark this answers is not in the document]
Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml]: Fixed

**Pull request merged**